### PR TITLE
Oktav2 - Unimplemented argument fix

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
relates: demisto/etc#32208

## Description
- *Breaking changes* - Fixed a typo in the **postalAddress** argument name in the ***okta-update-user*** command.

### CODE REVIEW

**Review by bakatzir — APPROVED** [review]
comment in the PR when all is green and you need a force merge.

**Comment by Bargenish** [comment]
@bakatzir - the PR is ready for merge. Thank you!

File: Packs/Okta/ReleaseNotes/2_1_3.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Okta v2
+- *Breaking changes* - Fixed a typo in the **postalAddress** argument name in the ***okta-update-user*** command.
+- Updated the Docker image from to *demisto/python3:3.9.1.14969*.
```
Comment (bakatzir) [inline Packs/Okta/ReleaseNotes/2_1_3.md:5]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.9.1.14969*.
```

**Comment by Bargenish** [comment]
@bakatzir - The PR is ready. Thank you 🙂

**Pull request merged**